web/components/templates/requestsV2/costPill.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from "react";
+import ThemedModal from "../../shared/themed/themedModal";
+import Link from "next/link";
+import { clsx } from "../../shared/clsx";
+import { ArrowTopRightOnSquareIcon } from "@heroicons/react/24/outline";
+
+const CostPill = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleOpenModal = (
+    event: React.MouseEvent<HTMLSpanElement, MouseEvent>
+  ) => {
+    event.stopPropagation();
+    setIsOpen(true);
+  };
+
+  return (
+    <>
+      <span
```

Comment:
nit: span's should not have an onClick one them. Change this to a button and make sure the flex is inline.